project/static/css/styles.css

```
@@ -206,6 +489,64 @@ td, th {
   padding: 8px;
 }
 
+th {
+    font-weight: bold;
+}
+
 tr:nth-child(even) {
   background-color: #dddddd;
+}
+
+
+
+/********** Register and longin page **********/
+
+.register-bg, .login-bg {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.register-bg {
+    background: url(https://images.unsplash.com/photo-1531265726475-52ad60219627?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1368&q=80) no-repeat center center fixed;
+    background-size: cover;
+}
+
+.login-bg {
+    background: url(https://images.unsplash.com/photo-1578850355726-ee748b33dcd8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1267&q=80) no-repeat center center fixed;
+    background-size: cover;
+}
+
+.register-link {
+    margin-top: 12px;
+}
+
+
+
+/********** Profile page **********/
+
+.cover {
+    width: 60vw;
+    background-color: var(--accent-color);
+}
+
+img {
+    max-width: 900px;
+}
+
+.account {
+    width: 100%;
+    padding: 3rem;
+    max-width: 680px;
+}
+
+.actions {
+    display: flex;
+    align-items: center;
+}
+
+.edit, .delete {
```

```suggestion
.edit, .delete .restore {
```
Restore button does not have margin